src/widgets/dropdown/lv_dropdown.c

```diff
@@ -773,7 +773,7 @@ static void lv_dropdown_event(const lv_obj_class_t * class_p, lv_event_t * e)
             }
             else if(dropdown->sel_opt_id + 1 < dropdown->option_cnt) {
                 dropdown->sel_opt_id++;
-                position_to_selected(obj);
+                position_to_selected(obj, LV_ANIM_OFF);
```

Comment:
It's my personal opinion that that one should be `LV_ANIM_ON` because key navigation is not too dissimilar to encoder navigation.

Same for the other key event.